packages/core/src/ux-configuration.ts

```diff
@@ -16,7 +16,7 @@ export class UXConfiguration {
   }
 
   public cssNormalize() {
-    PLATFORM.moduleName('./styles/normalize.css');
+    PLATFORM.moduleName('!css-loader!./styles/normalize.css');
```

Comment:
@3cp will this affect cli bundler in anyway? We need to prepend `!css-loader!` to make sure webpack does not extract the css (the starting `!`). Though I'm thinking maybe we just need the check below...
```ts
if (text && typeof (text) === 'string') {
```

Reply:
css-loader is definitely should not be there, because it's webpack only.

I understand there is some complication to load css dependency inside js/ts file, because our standard webpack config only does skip style-loader for css dep inside html file.

```
{
        test: /\.css$/i,
        issuer: [{ not: [{ test: /\.html$/i }] }],
        use: extractCss ? [{
          loader: MiniCssExtractPlugin.loader
        }, ...cssRules
        ] : ['style-loader', ...cssRules]
      },
      {
        test: /\.css$/i,
        issuer: [{ test: /\.html$/i }],
        // CSS required in templates cannot be extracted safely
        // because Aurelia would try to require it again in runtime
        use: cssRules
      },
```

There are two options you can get away from this:
1. don't use css file, use an inline string, then use `DOM.injectStyles(cssString)` to load the style.
2. use `<require from="./style.css"></require>` in a html file.

This is necessary to support unknown bundlers, although there are currently only webpack, cli-bundler, and dumber can work with Aurelia 1.

Reply:
@3cp Would `import 'normalize.vss';` work?

Reply:
Yes you are right, it should, aurelia-dialog does that.
We just need a dist build that cli-bundler can understand, cli-bundler doesn't understand `import()` (due to esprima). (BTW, dumber has no problem to support `import()`)

aurelia-dialog does that with an umd build, pointed by `"browser"` field in package.json.

Reply:
But honestly, I am not sure how webpack deals with dep in `import()`. Not sure how it skipped `style-loader`.

Reply:
Sorry, I totally misread your question 😆 

Yes, `import 'normalize.css'` should work, just leave it to bundler to load the style.

Reply:
I don't know how ux supports style customisation. But `import 'normalize.css'` statically load the style, the `DOM.injectStyle()` used in aurelia-dialog allow a delayed injection of css, possibly customised by user.